### DISCUSSION TITLE
ERROR

### DISCUSSION THREAD

**Discussion by idemudia314** [psycopg/psycopg2#1323] [Q&A]
i tried running 'pipenv install psycopg2' and it keeps showing the below error

ERROR: Could not find a version that satisfies the requirement psycopg2
ERROR: No matching distribution found for psycopg2

This is likely caused by a bug in psycopg2. Report this to its maintainers.

how do i solve this?

**Comment by dvarrazzo** [comment]
Ask pipenv. this is not our bug, despite what they say.

**Comment by dmwelch** [comment]
@idemudia314 Can you please re-title your discussion to something other than "ERROR"?
